### ISSUE TITLE
[Wasm] InvalidCastException when setting a <Path>'s Data as string from resources

### ISSUE THREAD

**Issue by carldebilly** [unoplatform/uno#7576]
### Current behavior

https://playground-wasm.azurewebsites.net/#ac616968

Following XAML in WASM
``` xml
<StackPanel VerticalAlignment="Center"
            HorizontalAlignment="Center">
            <StackPanel.Resources>
                <x:String x:Key="MyIcon">M1 3.5A1.5 1.5 0 0 1 2.5 2h2.764c.958 0 1.76.56 2.311 1.184C7.985 3.648 8.48 4 9 4h4.5A1.5 1.5 0 0 1 15 5.5v7a1.5 1.5 0 0 1-1.5 1.5h-11A1.5 1.5 0 0 1 1 12.5v-9zM2.5 3a.5.5 0 0 0-.5.5V6h12v-.5a.5.5 0 0 0-.5-.5H9c-.964 0-1.71-.629-2.174-1.154C6.374 3.334 5.82 3 5.264 3H2.5zM14 7H2v5.5a.5.5 0 0 0 .5.5h11a.5.5 0 0 0 .5-.5V7z</x:String>
            </StackPanel.Resources>

            <Path Data="{StaticResource MyIcon}"
                  Fill="#FF159BFF" 
                  Margin="0,26,0,0"/> 

</StackPanel>
```

When PathData is set as Resources using a string, it's generating an `InvalidCastException` at runtime.

### Expected behavior
Path data should be converted properly at runtime.

### How to reproduce it (as minimally and precisely as possible)
https://playground-wasm.azurewebsites.net/#ac616968

### Environment

WASM

**Comment by Youssef1313** [comment]
For whatever reason it's working locally on `master` branch.

![image](https://user-images.githubusercontent.com/31348972/144732711-a5e08fcb-4040-4394-b14a-fed183a644f8.png)

**Comment by jeromelaban** [comment]
The playground is not yet updated to the latest, and it's likely already fixed in the latest bits.

**Comment by jeromelaban** [comment]
Fixed in 4.7-dev
